Workspaces/ProjectWorkspace/Scripts/FolderListViewController.cs

```diff
@@ -140,7 +148,12 @@ static FolderData GetFolderDataByGUID(FolderData data, string guid)
 				{
 					var folder = GetFolderDataByGUID(child, guid);
 					if (folder != null)
+					{
+						if (folderToRootCallback != null)
+							folderToRootCallback(folder);
```

Comment:
Shouldn't this be called on child, not folder? For a > 3 level deep tree, we won't get the root folder to expand.